src/server/scripts/Custom/Paymaster.cpp

```diff
@@ -0,0 +1,255 @@
+#include "AccountMgr.h"
+#include "ScriptMgr.h"
+#include "Player.h"
+#include "Config.h"
+#include "Common.h"
+#include "Chat.h"
+#include "ObjectAccessor.h"
+#include "ObjectMgr.h"
+#include "Player.h"
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "ScriptedGossip.h"
+#include "SharedDefines.h"
+#include "World.h"
+#include "WorldSession.h"
+#include "DataStores/DBCStores.h"
+#include "Spells/SpellMgr.h"
```

Comment:
the folder is not needed
```suggestion
#include "SpellMgr.h"
```